paper/paper.md

```diff
@@ -43,29 +43,29 @@ Here are a few examples:
 - [`GPUE`](https://github.com/GPUE-group/GPUE) [@Schloss2018] for 1D to 3D Gross-Pitaevskii equations accelerated on GPU, in C++ (currently unmaintained).
 - [`py-fmas`](https://github.com/omelchert/py-fmas) for 1D NLSE in optical fibers, with a split-step method (currently unmaintained).
 
-With our project, we bring similar performance to C++ and Julia implementations, while striving for accessibility and maintainability by using the prevalant language in the physics community, Python.
+With our project, we bring similar performance to C++ and Julia implementations, while striving for accessibility and maintainability by using Python.
 Using an easy to extend object-oriented classes, users can readily input experimental parameters to quickly model real setups.
 
 # Functionality
 
-`NLSE` harnesses the power of pseudo-spectral schemes in order to solve efficiently the following general type of equation:
```

Comment:
I changed all "in order to" to "to". I use "in order to" a lot (I'm an Italian native speaker), but I've been told that it's too verbose for no reason and using "to" is better.